models/fixtures/user.yml

```diff
@@ -66,7 +66,7 @@
   num_followers: 2
   num_following: 1
   num_stars: 2
-  num_repos: 9
+  num_repos: 10
```

Comment:
Ditto: it seems like this number should be able to be auto-generated when the tests boot by running a GROUP BY on the `repository` table.

Reply:
But nope: the same bump had to happen in https://github.com/go-gitea/gitea/commit/36a2d2f919ee445f0374c210309b336e7c3c1f8e#diff-90061f7311f7bdfc66f8d58f6eb248d5abd734a2fe479896ab8cdea3fe46d4c8R1105